sdk/lib/3rdparty/stlport/CMakeLists.txt

```diff
@@ -53,5 +46,16 @@ if(USE_CLANG_CL)
     target_compile_options(stlport PRIVATE -Wno-tautological-unsigned-zero-compare)
 endif()
 
+target_include_directories(stlport PRIVATE ${REACTOS_SOURCE_DIR}/sdk/include/c++/stlport)
+target_compile_definitions(stlport PRIVATE
+    _STLP_USE_EXCEPTIONS _DLL __USE_CRTIMP
+    _BUILD_STLPORT NATIVE_CPP_INCLUDE=${REACTOS_SOURCE_DIR}/sdk/include/c++)
+
+target_include_directories(stlport INTERFACE "$<$<COMPILE_LANGUAGE:CXX>:${REACTOS_SOURCE_DIR}/sdk/include/c++/stlport>")
+target_compile_definitions(stlport INTERFACE "$<$<COMPILE_LANGUAGE:CXX>:NATIVE_CPP_INCLUDE=${REACTOS_SOURCE_DIR}/sdk/include/c++>")
+set_target_properties(stlport PROPERTIES
+    WITH_CXX_RTTI TRUE
+    WITH_CXX_EXCEPTIONS TRUE)
+
```

Comment:
Generic comment (applies to all other cmakelists as well):
This looks over-complicated; it would be nice to have a macro (like the original set_cpp) that hides all these details.

Reply:
Done